src/components/SignatureDishes/SignatureDishes.tsx

```diff
@@ -3,13 +3,17 @@ import { SectionTitle } from "../SectionTitle/SectionTitle";
 import styles from "./SignatureDishes.module.scss";
 import data from "../../data/data.json";
 import { Swiper, SwiperSlide } from "swiper/react";
-import goToIcon from "../../assets/images/goToIcon.svg";
 import DishCard from "../DishCard/DishCard";
 import useIsMobile from "../../hooks/useIsMobile";
 import useIsTablet from "../../hooks/useIsTablet";
 import GoToAllRestaurantsButton from "../GoToAllRestaurantsButton/GoToAllRestaurantsButton";
 
 const SignatureDishes = () => {
+  const mobileDishImageWidth = 245;
+  const mobileDishImageHeight = 152;
+  const desktopDishImageWidth = 380;
+  const desktopDishImageHeight = 306;
+
```

Comment:
These variables describe very specific use cases.
As we discussed, this can be more confusing than helpful.